specification/databricks/resource-manager/Microsoft.Databricks/stable/2024-05-01/accessconnector.json

```diff
@@ -0,0 +1,438 @@
+        "properties": {
+          "description": "Azure Databricks accessConnector properties",
+          "$ref": "#/definitions/AccessConnectorProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../preview/2022-04-01-preview/databricks.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Information about azure databricks accessConnector."
+    },
+    "AccessConnectorProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "Provisioning status of the accessConnector.",
+          "readOnly": true,
+          "enum": [
+            "Deleted",
+            "Failed",
+            "Succeeded"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "referedBy": {
+          "description": "List of workspaces referring this accessConnector.",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "AccessConnectorUpdate": {
+      "description": "An update to an azure databricks accessConnector.",
+      "type": "object",
+      "properties": {
+        "tags": {
+          "description": "Resource tags.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "identity": {
+          "$ref": "../../../../../common-types/resource-management/v3/managedidentity.json#/definitions/ManagedServiceIdentity"
+        }
+      }
+    },
+    "AccessConnectorListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AccessConnector"
+          },
+          "description": "The array of azure databricks accessConnector."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to use for getting the next set of results."
+        }
+      },
+      "description": "List of azure databricks accessConnector."
+    }
+  },
+  "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The ID of the target subscription."
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "The API version to use for this operation."
+    },
+    "ResourceGroupName": {
+      "name": "resourceGroupName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The name of the resource group. The name is case insensitive.",
+      "pattern": "^[-\\w\\._\\(\\)]+$",
+      "minLength": 1,
+      "maxLength": 90,
+      "x-ms-parameter-location": "method"
+    },
```

Comment:
I'd recommend switching to use references to the common-types/v5/types.json parameter definitions

Reply:
Sure, probably we can take that as a separate PR and it. Not as part of the current feature work we are doing.

Reply:
My personal opinion is its better to just do it now and get it done with. But up to you.

Reply:
Hi @TimLovellSmith I have modified the parameters to reference to the parameters defined in the common-types v5. Could you please take a look at it again?

Reply:
You're signed off for ARM review now.
The examples should just need realistic subscription ids (or any other fixes) to satisfy ModelValidation before merge.